### PR TITLE
docs(QTM-368): updates documentation template

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-368

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Code review
- [ ] Documentation updated

### CODE REVIEW

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
[ ] Code review
[x] Documentation

File: stories/Checkbox/Checkbox.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+export const Checked = Template.bind({});
+Checked.args = {
+  checked: true,
+};
+
+export const Disabled = Template.bind({});
+Disabled.args = {
+  disabled: true,
+};
```
[the remark this answers is not in the document]
Comment (toncabral88) [reply stories/Checkbox/Checkbox.stories.jsx:15]: Good point. I'll fix this.

File: stories/Checkbox/CheckboxGroup.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  options.map(({ label, name, checked, disabled }) => (
+    <>
+      <CheckboxGroup.Checkbox
+        label={label}
+        name={name}
+        checked={checked}
+        disabled={disabled}
+        key={name}
+      />
+    </>
```
Comment (MarcosViniciusPC) [inline stories/Checkbox/CheckboxGroup.stories.jsx:40]: The key warning is still shown in the console. I suggest removing the fragment element or adding a key to it

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code Review
[x] Documentation

File: stories/Checkbox/CheckboxGroup.stories.jsx (hunk 2 above)
Comment (toncabral88) [reply stories/Checkbox/CheckboxGroup.stories.jsx]: It's more easier understand suggestion when it comes with a description

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**